### DISCUSSION TITLE
Markdowning in issues has a bug when numbered list is used

### DISCUSSION THREAD

**Discussion by fxdeniz** [community/community#20552] [Projects and Issues] [answered]
I encountered this problem when I was editing an issue. It happens when code tag is used at the end of numbered list. Pressing enter for new line causes arbitrary number of list numbers being added.

I'm also adding picture and video to explain situation better:

![github](https://user-images.githubusercontent.com/104620840/179065735-e4d4a1de-f3fc-448f-aafa-89bfce7ecd8f.png)

video (in gif format):

![ezgif-3-b52219bc7d](https://user-images.githubusercontent.com/104620840/179067025-7b8e623c-072b-4923-9602-18c9d18f0d39.gif)

**Comment by oMvhmoud** [comment, marked as answer]
Not working for me, I guess fixed.
![Recording 2022-07-15 at 15 03 15](https://user-images.githubusercontent.com/40049200/179228550-8d0ed90b-915c-44ff-84fa-447788c0ef92.gif)
